Src/java/cqf-fhir-npm/build.gradle

```diff
@@ -3,4 +3,6 @@
 dependencies {
     implementation project(':cql-to-elm')
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.9.1'
+    implementation group: 'org.apache.commons', name: 'commons-compress', version: '1.20'
```

Comment:
I don't see where you are using these dependencies

Reply:
Oops, missed that during cleanup